types/is-html/index.d.ts

```diff
@@ -0,0 +1,11 @@
+// Type definitions for is-html 2.0
+// Project: https://github.com/sindresorhus/is-html
+// Definitions by: Samuel Menigat <https://github.com/smenigat>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+/**
+ * Check if given input string is HTML
+ * @param input The input string
+ */
+declare function isHTML(input?: string): boolean;
```

Comment:
```suggestion
declare function isHTML(input: string): boolean;
```
I know implementation supports `whatever` via `test` methods details and runtime behaviour, but was that an intent?
> Check if given input string is HTML
(L.7)

Reply:
I'm fine either way, but if you keep it, please change to `input: string | undefined` because passing zero arguments doesn't make sense.